src/ProjectTemplates/test/Helpers/Project.cs

```diff
@@ -121,7 +123,7 @@ internal async Task<ProcessEx> RunDotNetPublishAsync(bool takeNodeLock = false,
             await effectiveLock.WaitAsync();
             try
             {
-                var result = ProcessEx.Run(Output, TemplateOutputDir, DotNetMuxer.MuxerPathOrDefault(), $"publish -c Release /bl", packageOptions);
+                var result = ProcessEx.Run(Output, TemplateOutputDir, DotNetMuxer.MuxerPathOrDefault(), $"publish -c Release /bl /nr:false {additionalArgs}", packageOptions);
```

Comment:
Turning off node reuse avoids having `msbuild` processes hanging around after builds completed.

Reply:
Can we not do this? We want these tests to match the customer experience as close as possible. We already have a step that kills all the left-over processes after the build/test runs.

Reply:
Where does this happen? When I ran these tests locally, I saw several dotnet processes sticking around and had to manually kill it between re-runs. Also, I would think the CTI tests are a better approximation for testing what a user experience looks like. These are meant to verify if the templates run and build, changing how how msbuild runs doesn't change the outcome.

Reply:
The CI does this. I'm simply worried that this adds instability to the template tests.